src/check_ast.rs

```diff
@@ -689,13 +689,20 @@ where
                 }
                 ExprContext::Del => self.handle_node_delete(expr),
             },
-            ExprKind::Call { func, .. } => {
+            ExprKind::Call { func, args, .. } => {
                 if self.settings.select.contains(&CheckCode::R002) {
                     if let Some(check) = checks::check_assert_equals(func, self.autofix) {
                         self.checks.push(check)
                     }
                 }
 
+                // flake8-super
+                if self.settings.select.contains(&CheckCode::SPR001) {
```

Comment:
Should we validate that the current scope stack is function, then class?

Reply:
Eh, maybe non-critical.

Reply:
This is debatable. `flake8-super` does not that. But, `pyugrade` does.

In my opinion, it is not required. Because `super` cannot be redefined due to `flake8-builtins` rule. I also don't know any real-life use-cases where `super` is calledd somewhere outside of methods.

So, probably it is up to you.